proto/lbm/collection/v1/event.proto

```diff
@@ -299,3 +299,23 @@ message EventDetached {
   // token being detached.
   string subject = 4;
 }
+
+// EventOwnerChanged is emitted when the owner of token is changed by operation applied to its ancestor.
+//
+// Since: 0.46.0 (finschia)
+message EventOwnerChanged {
```

Comment:
Is there no changed owner information in this event?

Reply:
Yes, owner was not included in the event on Daphne.

Reply:
If so, can I understand that it was not added for backwards compatibility?

Reply:
Adding a new field would not break the compatibility, but there is no reason to add it.